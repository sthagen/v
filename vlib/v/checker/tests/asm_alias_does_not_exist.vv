asm amd64 {
	mov ebx, a
}
