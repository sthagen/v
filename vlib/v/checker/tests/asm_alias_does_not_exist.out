vlib/v/checker/tests/asm_alias_does_not_exist.vv:2:11: error: alias or label `a` does not exist
    1 | asm amd64 {
    2 |     mov ebx, a
      |              ^
    3 | }
